packages/core/src/pages/index.tsx

```diff
@@ -31,6 +31,14 @@ function Page({
   }
 
   const publisherId = settings?.seo?.publisherId ?? storeConfig.seo.publisherId
+  const orgAddress = settings?.seo?.organization?.address
+
+  const address =
+    orgAddress && Boolean(Object.values(orgAddress).find(Boolean))
+      ? Object.fromEntries(
+          Object.entries(orgAddress).filter(([, value]) => Boolean(value))
+        )
+      : null
```

Comment:
suggestion: I think this logic got a little complicated to understand.. What do you think about using reduce and omitting the conditional `Boolean(Object.values(orgAddress).find(Boolean))`? 
Something like:
```suggestion
  const organizationAddress = Object.entries(
    settings?.seo?.organization?.address ?? {}
  ).reduce(
    (acc, [key, value]) => {
      if (value) {
        acc[key] = value
      }
      return acc
    },
    {} as Record<string, string>
  )
```
And on the `OrganizationJsonLd` check if there are keys on this object: `Object.keys(organizationAddress).length !== 0`

Reply:
agreed! ⭐